src/content/self-hosted/helm-configuration.mdx

```diff
@@ -485,6 +485,35 @@ regcredsManager:
       memory: 800Mi
 ```
 
+### resourceManager
+
+Use this section to configure the [Resource Manager](admin/resource-manager.mdx)
+
+- `enabled`: When set to `true`, the Resource Manager is installed. When Resource Manager is installed, Okteto creates a CronJob responsible for computing the cpu and memory recommendations for your cluster.
```

Comment:
> Instead of "for your cluster", I would say "for your pods"

@pchico83 I've updated the copy. 

> I would say "for your pods" and I would be explicit about that the recommendation is for cpu/memory requests based on real time utilization using the Metrics API

If we document that in the dedicated page for Resource Manager, we can keep this section only dedicated to the actual helm config